--- boost/detail/basic_pointerbuf.hpp	2018-07-05 11:11:59.682145892 +1200
+++ boost/detail/basic_pointerbuf.hpp	2018-07-05 11:12:32.053562744 +1200
@@ -39,7 +39,7 @@
    typedef typename base_type::off_type off_type;
 
 public:
-   basic_pointerbuf() : base_type() { setbuf(0, 0); }
+   basic_pointerbuf() : base_type() { this_type::setbuf(0, 0); }
    const charT* getnext() { return this->gptr(); }
 
 #ifndef BOOST_NO_USING_TEMPLATE
