--- /usr/include/boost/serialization/unordered_collections_save_imp.hpp
+++ /usr/include/boost/serialization/unordered_collections_save_imp.hpp
@@ -25,6 +25,7 @@
 #include <boost/serialization/version.hpp>
 #include <boost/serialization/collection_size_type.hpp>
 #include <boost/serialization/item_version_type.hpp>
+#include <boost/serialization/library_version_type.hpp>
 
 namespace boost{
 namespace serialization {
